Sky Atlas/Sky Atlas.html

```diff
@@ -591,11 +891,13 @@ <h1>Articles</h1>
           <td>Distribution Of Agent Tokens</td>
           <td>Article</td>
           <td>
-            To the extent Sky invests capital in Agents and receives tokens in exchange, it will distribute those tokens to the following stakeholders:
+            To the extent Sky invests capital in Agents and receives tokens in exchange, it will distribute those tokens to the following
+            stakeholders:
```

Comment:
Don't add line breaks that aren't needed.

```suggestion
            To the extent Sky invests capital in Agents and receives tokens in exchange, it will distribute those tokens to the following stakeholders:
```